packages/core/core/src/Transformation.js

```diff
@@ -250,13 +251,19 @@ export default class Transformation {
           moduleName,
           parcelConfig.resolvedPath
         );
```

Comment:
So, if I understand correctly, all config for all plugins is loaded in the main thread. Then it's sent back to the transformer thread, and rehydrated. But, if it needs to be reloaded, it is immediately reloaded again on the transformer thread. So we end up needing to load it twice in that case?

Reply:
Yeah, for babel that will happen if you are using a `babel.config.js` file and you have something like:
```
module.exports = {plugins: [require('babel-plugin-whatever')]}
```
because at that point all we have is a function which can not be rehydrated. And unfortunately, we don't know if the user is using this type of configuration until we try to load it. Not ideal obviously, but I thought it would be better to just support it and surface a warning that this type of configuration has performance impacts. I was thinking it would also be a good idea to document how to optimize for caching in a README for this package. I'll add that as a TODO.

Reply:
Do configs need to be loaded on the main thread at all? I guess to check the graph? What if we just didn't store anything in the graph at all for configs that are uncacheable?

Reply:
We probably need some placeholder so we can at least invalidate on startup. Might not have to store an entire Config class though.